tests/integration-tests/test-suites/initiateDataRequestNegativeTests.spec.ts

```diff
@@ -3,112 +3,81 @@ import {
 import { getEnv } from '../../shared-test-code/utils/helpers'
 
-import { zendeskConstants } from '../../shared-test-code/constants/zendeskParameters'
-import { setCustomFieldValueForRequest } from '../../shared-test-code/utils/zendesk/generateZendeskTicketData'
 import { cloudwatchLogFilters } from '../constants/cloudWatchLogfilters'
-import { requestConstants } from '../constants/requests'
-
-const closeTicketComment =
-  'Your ticket has been closed because some fields were invalid. ' +
-  'Here is the list of what was wrong: From Date is in the future, To Date is in the future'
+import { getWebhookRequestDataForTestCaseNumberAndDate } from '../utils/getWebhookRequestDataForTestCaseNumberAndDate'
+import { sendWebhookRequest } from '../../shared-test-code/utils/zendesk/sendWebhookRequest'
 
 describe('Invalid requests should not start a data copy', () => {
-  describe('invalid recipient email', () => {
+  describe('invalid request - invalid recipient email', () => {
     let ticketId: string
 
     beforeEach(async () => {
-      const ticketData = { ...requestConstants.valid }
-
-      setCustomFieldValueForRequest(
-        ticketData,
-        zendeskConstants.fieldIds.recipientEmail,
-        'invalid@test.gov.uk'
-      )
-
-      ticketId = await createZendeskTicket(ticketData)
-      await approveZendeskTicket(ticketId)
+      const defaultWebhookRequestData =
+        getWebhookRequestDataForTestCaseNumberAndDate(1, '2022-01-01')
+      defaultWebhookRequestData.recipientEmail =
+        'txma-team2-bogus-ticf-analyst-dev@test.gov.uk'
+      ticketId = defaultWebhookRequestData.zendeskId
+      await sendWebhookRequest(defaultWebhookRequestData)
     })
 
-    afterEach(async () => {
-      await deleteZendeskTicket(ticketId)
-      console.log(
-        'recipient email not in approved list should not start data retrieval process, and should close ticket test ended'
-      )
-    })
-
-    test('recipient email not in approved list should not start data retrieval process, and should close ticket', async () => {
-      console.log(
-        'recipient email not in approved list should not start data retrieval process, and should close ticket test started'
-      )
-
+    it('recipient email not in approved list should not start data retrieval process', async () => {
       const initiateDataRequestEvents =
         await getCloudWatchLogEventsGroupByMessagePattern(
           getEnv('INITIATE_DATA_REQUEST_LAMBDA_LOG_GROUP_NAME'),
           [cloudwatchLogFilters.webhookReceived, 'zendeskId', `${ticketId}\\\\`]
         )
 
-      assertEventPresent(
+      const isWebhookInvalidMessageInLogs = assertEventPresent(
         initiateDataRequestEvents,
         cloudwatchLogFilters.webhookInvalid
       )
-      assertEventNotPresent(
+      expect(isWebhookInvalidMessageInLogs).toBe(true)
+
+      const isDataSentToQueueMessageNotInLogs = assertEventNotPresent(
```

Comment:
Do we need an `assertEventNotPresent` function now that `assertEventPresent` returns a boolean?